terraform/cp4data/terraform.tfvars

```diff
@@ -7,14 +7,16 @@
 // RECOMMENDED: to set the cluster_id using an external input, like this:
 // export TF_VAR_cluster_id="*********************"
 
+// The available Cloud Pak for Data versions are: 3.0 and 3.5
+install_version = 3.5
+
 // Otherwise, create a cluster based on values below
-project_name = "cloud-pack"
-owner        = "anonymous"
-environment  = "sandbox"
-region       = "us-south"
-// Using Humio account by default.  This resource group exists only on this account
+project_name   = "cloud-pack"
+owner          = "anonymous"
+environment    = "sandbox"
+region         = "us-south"
 resource_group = "cloud-pak-sandbox"
-// resource_group = "cloud-pak-sandbox"
+// resource_group = "cloud-pak-sandbox-ibm"
```

Comment:
Add these lines above 
`resource_group = "cloud-pak-sandbox"` like in `cp4app/terraform.tfvars`.  It makes things clearer.

```
// If using Humio account:
// resource_group = "cloud-pak-sandbox-ibm"
// If using standard partner account:
```